htdocs/core/class/commondocgenerator.class.php

```diff
@@ -541,7 +541,7 @@ public function get_substitutionarray_lines($line, $outputlangs, $linenumber = 0
 			'line_up_locale'=>price($line->subprice, 0, $outputlangs),
 			'line_total_up'=>price2num($line->subprice * $line->qty),
 			'line_total_up_locale'=>price($line->subprice * $line->qty, 0, $outputlangs),
-			'line_qty'=>$line->qty,
+			'line_qty'=>price($line->qty, 0, '', 0, 0), // Yes, it is a quantity, not a price, but we just want the formating role of function price
```

Comment:
line_qty must be the brut value, so for example 123456789,12345
When you say you want the formatting of function price. Do you mean you want the locale formatting (with space between thousand for example ?). If yes, you must introduce another tag
'line_qty_locale' and price function should have $outputlangs as parameter.